editor/index.js

```diff
@@ -83,6 +84,7 @@ export function recreateEditorInstance( target, settings ) {
 export function createEditorInstance( id, post, settings ) {
 	const target = document.getElementById( id );
 	const reboot = recreateEditorInstance.bind( null, target, settings );
+	shimMediaEditor( store );
```

Comment:
Should we pass `store` as an argument here, or could the shim file itself just import from `store` instead? Wondering more for consistency's sake, since if it's available otherwise and more direct, people will use it that way.